<launch>
  <arg name="camera_config" default="$(find scooping_cv)/config/camera.yaml"/>
  <arg name="filter_config" default="$(find scooping_cv)/config/particle_tracker.yaml"/>
  <arg name="service_config" default="$(find scooping)/config/object_tracker_services.yaml"/>

  <node
    name="multi_object_tracker_service"
    type="multi_object_tracker_service"
    pkg="scooping_cv"
    output="screen"
    respawn="false">
    <!-- launch-prefix="xterm -bg FloralWhite -fg MistyRose4 -e" -->
        <rosparam command="load" file="$(arg filter_config)" />
        <rosparam command="load" file="$(arg camera_config)" />
        <rosparam command="load" file="$(arg service_config)" />
  </node>
</launch>
